Makefile.toml

```diff
@@ -142,42 +141,22 @@ run_task = "format"
 
 [tasks.default]
 clear = true
+script_runner = "@duckscript"
 script = ['''
     echo
-    echo "Main tasks:"
-    echo "- generate-api: Generates Elasticsearch client from REST API specs"
-    echo "- start-elasticsearch: Starts Elasticsearch docker container with the given version and distribution"
-    echo "- stop-elasticsearch: Stops Elasticsearch docker container, if running"
-    echo "- test-yaml: Generates and runs yaml_test_runner package xpack/oss tests against a given Elasticsearch version"
-    echo "- test-generator: Generates and runs api_generator package tests"
-    echo "- test: Runs elasticsearch package tests against a given Elasticsearch version"
-    echo
-    echo "Most tasks use these environment variables:"
-    echo "- STACK_VERSION (default '$STACK_VERSION'): the version of Elasticsearch"
-    echo "- TEST_SUITE ('oss' or 'xpack', default '$TEST_SUITE'): the distribution of Elasticsearch"
-    echo "- CI (default not set): set when running on CI to determine whether to start Elasticsearch and format test output as JSON"
-    echo
-    echo "Run 'cargo make --list-all-steps' for a complete list of available tasks."
-    echo
-''']
-
-[tasks.default.windows]
-script = ['''
-    @echo off
-    echo.
     echo Main tasks:
     echo - generate-api: Generates Elasticsearch client from REST API specs
     echo - start-elasticsearch: Starts Elasticsearch docker container with the given version and distribution
     echo - stop-elasticsearch: Stops Elasticsearch docker container, if running
     echo - test-yaml: Generates and runs yaml_test_runner package xpack/oss tests against a given Elasticsearch version
     echo - test-generator: Generates and runs api_generator package tests
     echo - test: Runs elasticsearch package tests against a given Elasticsearch version
-    echo.
+    echo
     echo Most tasks use these environment variables:
     echo - STACK_VERSION (default '$STACK_VERSION'): the version of Elasticsearch
     echo - TEST_SUITE ('oss' or 'xpack', default '$TEST_SUITE'): the distribution of Elasticsearch
-    echo - CI (default not set): set when running on CI to determine whether to start Elasticsearch and format test output as JSON
-    echo.
+    echo - CARGO_MAKE_CI (default not set): set when running on CI to determine whether to start Elasticsearch and format test output as JSON
```

Comment:
actually this should stay CI and not CARGO_MAKE_CI
its automatically set for you based on the env.